x/ecocredit/server/basket/invariants_test.go

```diff
@@ -1,4 +1,4 @@
-package basket_test
+package basket
```

Comment:
needed to set the module address within some of the tests, so had to switch the test package to access the unexported fields.